### PR TITLE
automate chain specs generation

### PR DESCRIPTION
**Pull request by raynaudoe**

### CODE REVIEW

File: .github/workflows/generate-chain-specs.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+      - name: Install Dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y git protobuf-compiler
+
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.79.0
```
Comment (al3mart) [inline .github/workflows/generate-chain-specs.yaml:35]: Let's use 1.77

File: .github/workflows/generate-chain-specs.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          target: wasm32-unknown-unknown
+          components: rust-src
+          override: true
+
+      - name: Add rust-src
+        run: rustup component add rust-src --toolchain 1.79.0-x86_64-unknown-linux-gnu
```
Comment (al3mart) [inline .github/workflows/generate-chain-specs.yaml:41]: Same than above. Let's use 1.77 as the [fellowship does](https://github.com/polkadot-fellows/runtimes/releases/tag/v1.3.2).

File: .github/workflows/generate-chain-specs.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      - name: Generate chain specs
+        run: |
+          mkdir -p chain-specs/local
+          PACKAGES=(
+            "paseo-local"
+            "paseo-dev"
+            "asset-hub-paseo-local"
+            "bridge-hub-paseo-local"
+            "people-paseo-local"
+            "coretime-paseo-local"
```
Comment (raynaudoe) [inline .github/workflows/generate-chain-specs.yaml:59]: These ones will be removed

File: .github/workflows/generate-chain-specs.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            "coretime-paseo-tot"
+            "coretime-paseo"
```
Comment (al3mart) [inline .github/workflows/generate-chain-specs.yaml:61]: These two can be removed.

**Review by al3mart — CHANGES_REQUESTED** [review]
Left a comment with a small nit. Aside from that it looks good.

File: .github/workflows/generate-chain-specs.yaml (hunk 4 above)
Comment (raynaudoe) [reply .github/workflows/generate-chain-specs.yaml]: Done!

**Review by al3mart — APPROVED** [review]

**Pull request merged**